src/init.cpp

```diff
@@ -690,13 +690,13 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
 #endif
 
     if (!SetupNetworking())
-        return InitError("Error: Initializing networking failed");
+        return InitError("Initializing networking failed");
```

Comment:
Otherwise it's an `Error: Error:` overkill...

**After** this commit you get:

```
$ src/bitcoind -regtest 
Error: Initializing networking failed
```

or 

![screenshot from 2015-10-22 19-42-54](https://cloud.githubusercontent.com/assets/6399679/10673290/488400ba-78f5-11e5-8dc0-079969b9fdd4.png)